.agent/scripts/pr.py

```
@@ -1430,38 +1508,447 @@ def generate_diff_stats(files: list[tuple[str, str]]) -> str:
+                f"repos/{owner}/{repo}/pulls/{pr_number}/files",
+            ]
+        )
+
+        files_data = json.loads(result.stdout)
+        if not files_data:
+            return DiffError(
+                message="No files returned from API",
+                source=source,
+                is_too_large=False,
+            )
+        return files_data
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)